### PR TITLE
add quest details activity

### PR DESCRIPTION
**Pull request by isabelleleeson**
Go to new activity with quest details when user clicks on quest 
Will add ability to edit/delete quest later
Also data still hard-coded since don't have requests to server yet
image

### CODE REVIEW

File: android/app/src/main/res/layout/activity_parent_quest_details.xml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <ImageView
+            android:id="@+id/quest_image_view"
+            android:src="@drawable/broom"
```
Comment (isabelleleeson) [inline android/app/src/main/res/layout/activity_parent_quest_details.xml]: I'll remove all this hard-coded data once requests to server are made for quests

File: android/app/src/main/java/com/chorestory/app/ChoreStoryActivity.java [hunk 2]
```diff
@@ -49,7 +49,7 @@ protected void navigateTo(Class<?> cls, String key, String value) {
         startActivity(intent);
     }
 
-    protected void navigateTo(Class<?> cls) {
+    public void navigateTo(Class<?> cls) {
```
Comment (isabelleleeson) [inline android/app/src/main/java/com/chorestory/app/ChoreStoryActivity.java]: had to make this public because I'm using it in a different package

**Comment by keller00** [comment]
Quests don't have pictures

**Comment by keller00** [comment]
There are no optional quests on the back-end

**Comment by isabelleleeson** [comment]
we're gonna remove optional/mandatory for now and keep quest images on front-end only using a quest_name => quest_image map

File: android/app/src/main/res/layout/activity_parent_quest_details.xml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            android:layout_height="wrap_content"
+            android:layout_below="@id/quest_owner_text_view"
+            android:layout_marginTop="20dp"
+            android:paddingStart="25dp"
+            android:paddingEnd="25dp"
+            android:text="Description: blablabla"
+            android:textSize="20sp" />
+
+    </RelativeLayout>
+</ScrollView>
```
Comment (jimzhaozhengwang) [inline android/app/src/main/res/layout/activity_parent_quest_details.xml]: add a new line

**Pull request merged**